cloudinit/settings.py

```diff
@@ -15,6 +15,7 @@
 CLOUD_CONFIG = "/etc/cloud/cloud.cfg"
 
 CLEAN_RUNPARTS_DIR = "/etc/cloud/clean.d"
+HOTPLUG_ENABLED_FILE = "/etc/cloud/hotplug.enabled"
```

Comment:
James' comment move this to `/var/lib/cloud` as we don't want 'people' to see this as a knob that they should be setting/touching directly. It's an internal implementation of cloud-init.

Reply:
Done.